github-runner-manager/src/github_runner_manager/metrics/runner.py

```diff
@@ -68,7 +106,21 @@
     name="job_duration_seconds",
     documentation="Time taken in seconds for the job to be completed.",
     labelnames=[labels.FLAVOR],
-    buckets=[60, 60 * 5, 60 * 10, 60 * 20, 60 * 30, 60 * 60, 60 * 60 * 2, float("inf")],
+    buckets=[
+        MINUTE_IN_SECONDS,
+        MINUTE_IN_SECONDS * 5,
+        MINUTE_IN_SECONDS * 10,
+        MINUTE_IN_SECONDS * 20,
+        MINUTE_IN_SECONDS * 30,
+        MINUTE_IN_SECONDS * 60,
+        # hours
+        HOURS_IN_SECONDS * 2,
+        HOURS_IN_SECONDS * 4,
+        HOURS_IN_SECONDS * 6,
+        # days
+        DAYS_IN_SECONDS * 3,
```

Comment:
Sorry, just looked it up, it is 5 days: https://docs.github.com/en/actions/reference/limits